docs/index.rst

```diff
@@ -43,80 +43,185 @@ to help you create fantastic documentation for your project.
-   api/index
-   embed
 
-.. _feature-docs:
+
+Advanced features of Read the Docs
+----------------------------------
+
+Read the Docs offers many advanced features and options.
+Learn more about these integrations and how you can get the most
+out of your documentation and Read the Docs.
+
+* **Advanced project configuration**:
+  :doc:`subprojects` |
+  :doc:`Single version docs <single_version>` |
+  :doc:`Privacy levels <privacy>`
+
+* **Multi-language documentation**:
+  :doc:`Translations and localization <localization>`
+
+* **Redirects**:
+  :doc:`User defined redirects <user-defined-redirects>` |
+  :doc:`Automatic redirects <automatic-redirects>`
+
+* **Topic specific guides**:
+  :doc:`How-to guides <guides/index>`
+
+* **Extending Read the Docs**:
+  :doc:`REST API <api/index>` |
+  :doc:`embed`
 
 .. toctree::
    :maxdepth: 2
+   :hidden:
    :glob:
-   :caption: Feature Documentation
+   :caption: Advanced features
 
-   webhooks
-   badges
-   custom_domains
-   localization
-   vcs
    subprojects
-   conda
-   canonical
    single_version
    privacy
+
+   localization
+
    user-defined-redirects
    automatic-redirects
+
+   guides/index
+
+   api/index
+   embed
+
+   vcs
+   conda
+   canonical
    features/*
 
-.. _about-docs:
+
+The Read the Docs project and organization
+------------------------------------------
+
+Learn about Read the Docs, the project and the company,
+and find out how you can get involved and contribute to the development and success
+of Read the Docs and the larger software documentation ecosystem.
+
+* **Getting involved with Read the Docs**:
+  :doc:`Contributing <contribute>` |
+  :doc:`roadmap` |
+  :doc:`gsoc` |
+  :doc:`Code of conduct <code-of-conduct>`
+
+* **Policies & Process**:
+  :doc:`security` |
+  :doc:`Privacy policy <privacy-policy>` |
+  :doc:`DMCA takedown policy <dmca/index>` |
+  :doc:`Policy for abandoned projects <abandoned-projects>` |
+  :doc:`Release notes & changelog <changelog>`
+
+* **The people and philosophy behind Read the Docs**:
+  :doc:`Team <team>` |
+  :doc:`Open source philosophy <open-source-philosophy>` |
+  :doc:`Our story <story>`
+
+* **Financial and material support**:
+  :doc:`advertising/index` |
+  :doc:`Sponsors <sponsors>`
+
+* **Read the Docs for business**:
+  :doc:`Support and additional features <commercial/index>`
+
+* **Running your own version of Read the Docs**:
+  :doc:`Private installations <custom_installs/index>`
+
 
 .. toctree::
    :maxdepth: 1
+   :hidden:
    :caption: About Read the Docs
 
    contribute
    roadmap
-   team
    gsoc
    code-of-conduct
+
+   security
    privacy-policy
-   advertising/index
-   sponsors
+   dmca/index
+   abandoned-projects
+   changelog
+
+   team
    open-source-philosophy
    story
-   abandoned-projects
-   dmca/index
 
-.. _dev-docs:
+   advertising/index
+   sponsors
+
+   commercial/index
+
+   custom_installs/index
+
 
 .. toctree::
    :maxdepth: 1
-   :caption: Developer Documentation
```

Comment:
Other than the security docs and changelog which I moved out of this section, I think that the developer docs should **not** be on the front docs page. Instead, I propose expanding `docs/contribute` and putting this toctree into a section there. I also think all of these RST files should be under `docs/development`.

Reply:
Makes sense. We really need to create a git-mapped redirect app, to make creating redirects for ourselves simpler.